_docs-sources/intro/overview/josh-feedback.md

```diff
@@ -0,0 +1,66 @@
+---
+'title': "Josh's Feedback"
+---
+
+# Josh's Feedback
+
+- What's the vision for "tool fundamentals"?
+  - How does it compare to Set Up Your Environment / Installing Neccessary Tools?
+  - Should "Setup Your Tools" (formerly Tool Fundamentals) be part of Set Up Your Environment?
+- "Next Steps" at bottom should be the same boldness. 
+- Should "Deploy Your First Module" focus on teaching The Gruntwork Way? 
```

Comment:
This is basically Pete's Zero2Hero by another name, which I understand to cover exactly that idea, in "Hello, world" fashion, by walking folks through deploying a single specific module. I don't think this one needs to be use case driven, at least at first, as it's not targeting a production deployment.